### ISSUE TITLE
Host Site

### ISSUE THREAD

**Issue by mariaalouisaa** [Vasystus/TechHive#6]
- Visit your website at a publicly accessible domain name.
The site should be hosted on GitHub Pages.

**Comment by vasystus** [comment]
So, If that's it, I'll deploy the site!

**Comment by mariaalouisaa** [comment]
Yes, go for it! 🥳

**Comment by vasystus** [comment]
Okay, it's alive!!! 
Looks a bit different from I saw it last time - the pictures are gone again( I saw them last push/merge) and padding around the sections a bit wide. And the fonts could be bigger? What do you see, what do you think?

**Comment by vasystus** [comment]
Could you push the finishing-touches again into the main branch?

**Comment by mariaalouisaa** [comment]
Yes, it looks different and images are not showing on the github pages, but they do on my local...
Also did you remove the lighter colour behind the "What we do"  as that has gone aswell.

I will try another push now.

**Comment by mariaalouisaa** [comment]
I just thought, maybe to get the images to show you need to click on the '/docs' option in github pages as they are in a folder in the repo? 

<img width="501" alt="Untitled2" src="https://user-images.githubusercontent.com/82417131/151065963-3b79954e-fdc7-4899-bc60-2d40becb6279.png">

**Comment by vasystus** [comment]
Tried that- deployment is failed.

**Comment by mariaalouisaa** [comment]
Hmm okay, maybe I should try taking the images out of the folder and
changing the src url. I will push the changes to later tonight / early in
the morning. We will get this to work!

**Comment by vasystus** [comment]
I merged/pushed the changes. Everything is visible and working now! 
I also added back the var(--lightest navy) to the :root- it appeared to be gone, that's why one section and form lost their colors.
If I'll have the time tomorrow, I probably will increase the font-sizes some of <p> in welcome and what we do section. And padding on the sides could be smaller...

But overall, Congratulations, we made it!!!:tada::confetti_ball::wine_glass:
